src/components/Home.js

```diff
@@ -1,10 +1,39 @@
 import React from "react";
+import { NavLink } from "react-router-dom";
 import "./Home.css"
+import img1 from "../images/wineGlasses.jpeg"
+import featImg1 from "../images/goatCheese.jpg"
+import featImg2 from "../images/wineBottle.jpg"
```

Comment:
Are these the same images from the database? If so, we should be able to pull that information from the db instead of reimporting them.